scene/gui/text_edit.h

```diff
@@ -37,6 +37,15 @@
 #include "scene/main/timer.h"
 #include "scene/resources/syntax_highlighter.h"
 #include "scene/resources/text_paragraph.h"
+#include "core/object/object.h"
+
+class TextEditSearchResult : public Object {
+	GDCLASS(TextEditSearchResult, Object);
```

Comment:
Three's no need to add `GDCLASS` is you do not register it for the use from the scripts.